modules/ROOT/pages/verify-nfs.adoc

```diff
@@ -0,0 +1,187 @@
+If the mount fails, you do not have the required permissions. 
+
+== Validate NFS Performance
+
+Perform careful analysis of your environment, both from the client and server point of view for optimal NFS performance.
+
+These are the high level steps for validating NFS performance:
+
+. Perform some basic operations to test the performance and latency of your NFS server. Specifically, these operations will: +
+* Transfer data from the client (Anypoint Platform environment) to the NFS server 
+* Measure how long each transfer takes 
+. To validate how the Anypoint Platform handles NFS files transfers, `read` and `write` operations are tested, with both large (128MB) and small (80KB) files.
+. Use the command `time dd if=/dev/zero of=/mnt/home/<nameOfFile> bs=<blockSize> count=<amountOfBlocks>` to test performance of the NFS server while transferring files with: +
+* Names `<nameOfFile>` 
+* The block size and number of blocks in the transfer are `<blockSize>` and `<amountOfBlocks>`, respectively.
+
+=== Test Writing and Reading Large files on the NFS server
+
+These `write` and `read` tests are performed using large files:
+
+* File size: 128 MB
+* Block size: 4 KB
+* Number of blocks: 32768  
+
+During the time the tests are running, different files are generated (one file per test).
+
+. To start with the testing, create a directory in which to mount the NFS server: 
++
+----
+sudo su
+mkdir -p /mnt/home
+----
+. Mount the NFS server: 
++ 
+----
+mount -t nfs4 -o proto=tcp,port=2049 <nfs-server>:<path> /mnt/home/
+----
+. Perform writes (five files of 128 MB each) on the NFS server by running the following command: 
++
+----
+time for i in {1..5}; do dd if=/dev/zero of=/mnt/home/greatfile$i.test bs=4k count=32768; done
+----
++
+The command output is similar to the following:
++
+----
+[root@ip-10-1-1-97 ec2-user]# time for i in {1..5}; do dd if=/dev/zero of=/mnt/home/greatfile$i.test bs=4k count=32768; done
+32768+0 records in
+32768+0 records out
+134217728 bytes (134 MB) copied, 2.12805 s, 63.1 MB/s
+.
+.
+.
+real 0m8.378s
+user 0m0.034s
+sys 0m0.792s
+----
++
+The `real` parameter in the test output should be less than 15 seconds. 
+
+==== Validate read Performance
+
+. Umount the NFS server: 
++
+----
+umount /mnt/home
+----
+. Mount the NFS server again by running: 
++
+----
+mount -t nfs4 -o proto=tcp,port=2049 <nfs-server>:<path> /mnt/home/
+----
+. Perform reads from the NFS server by running the following command: 
++
+----
+time for i in {1..5}; do dd if=/mnt/home/greatfile$i.test of=/dev/null bs=4k; done
+----
++
+The `real` parameter in the test output should be less than 15 seconds.
+
+==== Clear the Caches 
+
+Umount the NFS server to clear out any caches:
+
+----
+umount /mnt/home
+----
+
+=== Test Writing and Reading Small Files on the NFS server
+
+Perform the same type of testing you ran on large files with smaller files.  In this case, the tests are performed with:
```

Comment:
"you ran" seems colloquial to me. Maybe "Test small files as you did large files (in step?). "In this case" isn't needed. are we saying these tests are automatically performed or is this an instruction?